inf-clojure.el

```diff
@@ -208,6 +209,21 @@ whichever process buffer you want to use.")
 
 (put 'inf-clojure-mode 'mode-class 'special)
 
+(defcustom inf-clojure-repl-flavor 'clojure
+  "Symbol to define your REPL flavor.
+The default flavor is 'clojure, 'lumo is the other supported
+one."
+  :type 'symbol
+  :options '(clojure lumo)
+  :group 'inf-clojure)
+
+(defun inf-clojure--flavor-setup ()
+  "Setup inf-clojure defcustoms depending on the choose flavor."
+  (pcase inf-clojure-repl-flavor
+    (lumo (progn (message "[inf-clojure] will switch to the Lumo flavor")
```

Comment:
Those messages are bit confusing as you won't really be switching to anything - you'll start a REPL of this type.